generated/nidcpower/nidcpower/session.py

```diff
@@ -7498,16 +7498,23 @@ def __init__(self, resource_name, channels=None, reset=False, options={}, indepe
 
             grpc_options (nidcpower.grpc_session_options.GrpcSessionOptions): MeasurementLink gRPC session options
 
+            driver_warning_event (nidcpower.DriverWarningEvent): Driver warning event which can be subscribed to, with a callback method.
```

Comment:
This is not the documentation for instance attributes. It's the documentation for the `__init__` method arguments, but the variable is not actually an argument.